src/objects/api/v1/openapi.yaml

```diff
@@ -92,7 +92,7 @@ paths:
           Data filtering expressions are comma-separated and are structured as follows:
           A valid parameter value has the form `key__operator__value`.
           `key` is the attribute name, `operator` is the comparison operator to be used and `value` is the attribute value.
-          Note: Values can be string or numeric. Dates are not supported.
+          Note: Values can be string, numeric, or dates (ISO format; YYYY-MM-DD).
```

Comment:
Just curious: what caused this change?

Reply:
the schema was not updated in one of the merged PRs (https://github.com/maykinmedia/objects-api/pull/223)

Reply:
the schema was not updated in one of the merged PRs (https://github.com/maykinmedia/objects-api/pull/223)